edi_oca/models/edi_exchange_record.py

```diff
@@ -198,13 +198,16 @@ def _set_file_content(
             output_string = bytes(output_string, encoding)
         self[field_name] = base64.b64encode(output_string)
 
-    def _get_file_content(self, field_name="exchange_file", binary=True):
+    def _get_file_content(
+        self, field_name="exchange_file", binary=True, as_bytes=False
+    ):
         """Handy method to not have to convert b64 back and forth."""
         self.ensure_one()
         if not self[field_name]:
             return ""
```

Comment:
For consistency, do we need to manage this case as well?
```suggestion
            return "" if not as_bytes else b""
```

Reply:
hmm I don't think we need it here.
This is only for sending. If you want to manipulate the file in other circumstances you can do it in custom code.